src/frappe.xml

```diff
@@ -24,37 +24,37 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
 -->
-<style-scheme id="catppuccin-frappe" _name="Catppuccin Frappe" version="1.0">
+<style-scheme id="catppuccin-frappe" _name="Catppuccin Frappé" version="1.0">
```

Comment:
Did you test loading the theme to make sure it doesn't break on the accented e?

Reply:
Fair enough - I'll revert for now as I haven't tested it.